assets/js/views/specific-select.jsx

```diff
@@ -232,21 +226,20 @@ class ProductSpecificSearchResultsDropdown extends React.Component {
 					classNames="components-button--transition"
 					timeout={ { exit: 700 } }
 				>
```

Comment:
Was the CSS transition removed on purpose? Previously when the text changed to "Added" the element would also fade to white.

For reference, this `CSSTransition` element adds the `components-button--transition-exit` class to `wc-products-list-card__content` for 700ms before the row is removed from the DOM. (That's how React handles CSS transitions)